go.mod

```diff
@@ -5,14 +5,14 @@ go 1.15
 require (
 	github.com/mailru/easyjson v0.7.1 // indirect
 	github.com/onsi/gomega v1.10.3
-	github.com/shipwright-io/build v0.5.2-0.20210715083206-5d8fb411a1eb
-	github.com/spf13/cobra v1.1.3
+	github.com/shipwright-io/build v0.5.2-0.20211006161104-3b546e7103b6
```

Comment:
you said in slack you were going to bump this to the newly minted v0.6.0, right @coreydaley ?

Reply:
ah .. I do see https://github.com/shipwright-io/cli/pull/51though that only touches the Makefile/install script

Reply:
good catch, updated.